zeenea/public-connector-sdk
# [PLT-886] Prepare for beta release

JIRA issue: https://zeenea.atlassian.net/browse/PLT-886

##### PR summary:

- Remove lastSuccessfulVersion
- Remove SourceDataProcess and friends
- Remove IDE warnings
- Remove unused class ConnectionCode
- Improve Javadoc coverage
- Rework README
- Bump version for Beta

##### QA summary:

- Look at the README here: https://github.com/zeenea/public-connector-sdk/blob/feature/PLT-886/prepare-beta-release/README.md
- Build javadoc with `./gradlew javadocjar`
- Open javadoc (IDEA allows opening `index.html` with the browser)
- Give your opinion! Is it understandable? What should be adjusted?

##### Checklist for Developer:

- [x] Summary part has been documented
- [ ] Reviewers have been requested
- [ ] Reviews & comments have been taken into consideration
- [ ] Commits have been reworded and history cleaned

##### Checklist for Reviewer

- [ ] Code has been reviewed, commented and changes have been requested
- [ ] Reviews & comments have been taken into consideration
